framework/saleor/utils/normalize.ts

```diff
@@ -24,60 +19,47 @@ const money = ({ amount, currency }: Money) => {
   }
 }
 
-const normalizeProductOptions = (options: ProductOption[]) => {
-  return options?.map(({ id, name: displayName, values }) => ({
-    __typename: 'MultipleChoiceOption',
-    id,
-    displayName,
-    // values: values.map((value) => {
-    //   let output: any = {
-    //     label: value,
-    //   }
-    //   if (displayName.match(/colou?r/gi)) {
-    //     output = {
-    //       ...output,
-    //       hexColors: [value],
-    //     }
-    //   }
-    //   return output
-    // })
-    values: [],
-  }))
-}
+const normalizeProductOptions = (options: ProductVariant[]) => {
+  const optionNames = options
+    .map((option) => {
+      // can a variant have multiple attributes?
```

Comment:
@patrys could you confirm this?

Reply:
A single variant can have many attributes, some can also have multiple values